test/spec/modules/parrableIdSystem_spec.js

```diff
@@ -83,7 +89,7 @@ function decodeBase64UrlSafe(encBase64) {
   return encBase64.replace(/[-_.]/g, (m) => DEC[m]);
 }
 
-describe('Parrable ID System', function() {
+describe.only('Parrable ID System', function() {
```

Comment:
This shouldn't make it into the final PR. I would suggest when you're staging your changes for a commit you can do `git add -p .` and git will prompt you for each hunk if you want to include it in your commit or not. I often exclude some lines from a commit like this.